python/semantic_kernel/connectors/memory/azure_cosmosdb/azure_cosmosdb_settings.py

```diff
@@ -2,7 +2,7 @@
 
 from typing import ClassVar
 
-from pydantic import Field, SecretStr
+from pydantic import Field, SecretStr, ConfigDict
```

Comment:
you need to run Ruff, to fix the import order

Reply:
done